srv/modules/runners/rescinded.py

```diff
@@ -68,6 +77,60 @@ def osds(cluster='ceph'):
             log.debug(data[minion])
     return list(set(_ids))
 
+
+def orphaned_hosts_list(cluster='ceph'):
+    """
+    List any orphaned host entries in the CRUSH map
+    """
+    search = "I@cluster:{} and I@roles:master".format(cluster)
+
+    local = salt.client.LocalClient()
+    data = local.cmd(search, 'osd.orphaned', [], tgt_type="compound")
+    orphaned_hosts = data[list(data.keys())[0]]
+    log.debug("rescinded.orphaned_hosts_list: osd.orphaned returned ->{}<-"
+              .format(orphaned_hosts))
+    assert isinstance(orphaned_hosts, list), "List of orphaned CRUSH host entries is not a list"
+    return orphaned_hosts
+
+
+def orphaned_hosts(cluster='ceph'):
+    """
+    Return true if there are any orphaned host entries in the CRUSH map
+    """
+    if len(orphaned_hosts_list()) > 0:
+        return True
+    else:
+        return False
+
+
+def nodes(cluster='ceph', **kwargs):
+    """
+    List the short hostnames of minions that are not storage nodes
+    (there should be only one, but handle multiple)
+    """
+
+    local = salt.client.LocalClient()
+
+    # Restrict search to this cluster
+    search = "I@cluster:{}".format(cluster)
+
+    pillar_data = local.cmd(search, 'pillar.items', [], tgt_type="compound")
+    _crushhosts_to_delete = []
+    _orphaned_hosts = orphaned_host_list()
```

Comment:
Note: could call `osd.orphaned` module from here and drop the `orphaned_host_list` runner.